### DISCUSSION TITLE
PnP search web part gives error each time after updating to newer version

### DISCUSSION THREAD

**Discussion by smadhira5275** [microsoft-search/pnp-modern-search#3716] [Q&A]
Dear Team,
After updating package from 4.10.2 to 4.11.0, PnP search web parts are not working in all our SharePoint sites. Page shows the below error after updating the package: Attached is the word document with screenshots (one image shows web part errors and other one is shows, PnP upgrade required at site level). This was not the case earlier.

[PnP-IssueScreen.docx](https://github.com/microsoft-search/pnp-modern-search/files/15127980/PnP-IssueScreen.docx)

This is not the behavior for other packages which are deploying at app catalog. Issue is happening only with PnP package. Kindly help me here ?

Thanks.

**Comment by wobba** [comment]
It helps to expand the error message so we see what it is. The typical resolution for this, which can happen to other SPFx solutions is to:

* Delete the sppkg from the appcatalog
* Upload the new sppkg

And if CDN is enabled, this error seems to occur more frequently in my experience. I always recommend updating new packages outside of working hours and preferable on weekends to avoid caching issues when updating.
